_posts/2023-09-12-sekai-guardians-of-the-kernel.md

```diff
@@ -0,0 +1,399 @@
+---
+layout: post
+current: post
+cover:  assets/sekai/zerodaytea/cover.png
+navigation: True
+title: "Guardians of the Kernel"
+date: 2023-09-12 02:00:00
+tags: [SekaiCTF, rev]
+class: post-template
+subclass: 'post'
+author: ZeroDayTea
+---
+
+Kernel can be a scary word. That's alright though because we have an SMT solver on our team
+
+## Problem Description
+
+It’s just a warmup but with another layer which is the kernel.
```

Comment:
what does this mean

Reply:
copied description from Sekai themselves